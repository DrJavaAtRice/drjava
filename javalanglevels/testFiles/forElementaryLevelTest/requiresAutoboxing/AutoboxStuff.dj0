class AutoboxStuff {

  int method() {
    Integer i = 3;
    double d = i;
    Object o = 6;
    return i;
  }

}
