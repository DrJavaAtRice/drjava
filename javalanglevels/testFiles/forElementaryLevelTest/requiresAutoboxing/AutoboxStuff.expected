// Language Level Converter line number map: dj*->java. Entries: 10
//     1->4         2->5         3->6         4->7         5->8         6->9         7->10        8->11   
//     9->12       10->13   
class AutoboxStuff {

  public int method() {
    final Integer i = 3;
    final double d = i;
    final Object o = 6;
    return i;
  }


  /** This method is automatically generated by the Language Level Converter. */
  public AutoboxStuff() {
    super();
  }

  /** This method is automatically generated by the Language Level Converter. */
  public java.lang.String toString() {
    return getClass().getName() + "(" + 
        ")";
  }

  /** This method is automatically generated by the Language Level Converter. */
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    else if ((o == null) || (! o.getClass().equals(getClass()))) {
      return false;
    }
    else {
      AutoboxStuff cast = ((AutoboxStuff) o);
      return true;
    }
  }

  /** This method is automatically generated by the Language Level Converter. */
  public int hashCode() {
    return getClass().hashCode();
  }
}
